.github/workflows/pythonapp.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         python-version: 3.8
     - name: Install dependencies
       run: |
-        pip install nose numpy scipy scikit-learn pandas joblib pyyaml
+        pip install nose numpy scipy scikit-learn pandas joblib pyyaml parameterized
```

Comment:
what is giving us this scikit-learn dependency? can it be removed?

Reply:
Yep, I think it can be removed

Reply:
Also, joblib can be removed.

Reply:
:+1: